### PR TITLE
Convert .net core 3.1 to .net 6.0

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
**Execute Build Pipeline  -** 
image

File: samples/msgext-link-unfurling-meeting/csharp/README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ This sample illustrates a common scenario where a user shares a link to a resour
       * Select **web**.
       * Enter the **redirect URI** for the app in the following format: 
       - https://token.botframework.com/.auth/web/redirect, 
-      - https://%ngrokDomain%.ngrok.io/AuthEnd
-      - https://%ngrokDomain%.ngrok.io/AuthStart This will be the page where a successful implicit grant flow will redirect the user.
+      - https://%ngrokDomain%.ngrok.io/auth-end
```
Comment (Pawan-MSFT) [inline samples/msgext-link-unfurling-meeting/csharp/README.md:91]: Is in Routing or page also need to required AuthEnd to auth-end and AuthStart to auth-start?

**Review by Pawan-MSFT — COMMENTED** [review]
Please confirm

File: samples/msgext-link-unfurling-meeting/csharp/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-link-unfurling-meeting/csharp/README.md:91]: No, We only need to create azure web Authentication.

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**